core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -681,6 +681,7 @@ public void onNext(Frame dockerFrame) {
             @Override
             public void close() throws IOException {
                 consumer.accept(OutputFrame.END);
+                super.close();
```

Comment:
@bsideup - it's a bit of an anticlimactic commit, but it looks to me as though calling through to `ResultCallbackTemplate`'s close method should automatically close the relevant stream. When testing, this seems to effect an appropriate chain of:
```
container closes -> anonymous inner ResultCallbackTemplate subclass closes -> ResultCallbackTemplate closes -> stream gets closed
```

Unless you've noticed anything I've missed..?